appengine/flexible/hello_world/Gemfile

```diff
@@ -19,9 +19,12 @@ gem "sinatra"
 gem "webrick", "~> 1.8"
 
 # [END gae_flex_quickstart_dependencies]
-ruby "~>3.2"
+ruby "~>3.4"
```

Comment:
Is there a flexible runtime for Ruby 3.4? I was under the impression that the runtimes transitioned to the app engine standard platform some time ago. (I don't see a Ruby version update in https://github.com/GoogleCloudPlatform/ruby-docker for example.)

Reply:
https://github.com/GoogleCloudPlatform/ruby-docker is (or, perhaps, was) the source for the GAE Flex runtime for Ruby. I guess the runtime is maintained elsewhere now.